install.sh

```diff
@@ -34,7 +34,7 @@ then
 fi
 xcode_ver_num=`echo $xcode_ver | sed 's/^Xcode \([0-9][0-9]*\)\.\([0-9][0-9]*\)/\1\2/'`
 xcode_ver_str=`echo $xcode_ver | sed 's/^Xcode //'`
-if [[ $xcode_ver_num -lt $XCODE_MIN_VERSION ]]
```

Comment:
I'd rather fix the sed substitution, which is not working right in this case.  My expectation was that line 35 would lop off / ignore anything beyond the first two numerics in an x.y.z.p.d.q representation, but apparently not.  Taking a look now.